### ISSUE TITLE
SVGs in the AssetLib package are broken

### ISSUE THREAD

**Issue by CuteMoonGod** [Tattomoosa/VisionCone3D#1]
The SVGs in the addon, as downloaded from the AssetLib, are currently broken and preventing the compilation of the scripts. They can't be loaded by either godot nor Inkscape.

Replacing them with the SVG files from github and reimporting fixes this, but still.

![grafik](https://github.com/user-attachments/assets/f633f7b1-c84f-41a5-b076-d8d9c1a7adfe)

**Comment by Tattomoosa** [comment]
Thanks for letting me know. It was an issue with using git-lfs to track image files. Fixed in recent commit. Releasing 0.2.0 shortly with performance improvements, bugfixes, *and* working icons!

I've confirmed the icons show up when downloading from github, but I will wait until the asset library update is published to check there before I close the issue.

Updating will require a manual change (sorry!) as VisionCone3D now inherits Area3D as part of other various performance improvements. Remove and replace in scene or use "Change Type...", see CHANGELOG.
